### PR TITLE
feat: gRPC query for operator and chain configuration

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #XXX

## What is the purpose of the change

Backport:
- [x] https://github.com/cosmos/cosmos-sdk/pull/13485
- [x] https://github.com/cosmos/cosmos-sdk/pull/13577

### CODE REVIEW

File: server/start.go [hunk 1]
```diff
@@ -289,6 +289,10 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 
 		app.RegisterTxService(clientCtx)
 		app.RegisterTendermintService(clientCtx)
+
+		if a, ok := app.(types.ApplicationQueryService); ok {
+			a.RegisterNodeService(clientCtx)
+		}
```
Comment (mattverse) [inline server/start.go:295]: whats a possible condition that app might not be implementing `ApplicationQueryService`?

**Review by mattverse — APPROVED** [review]
LGTM

File: server/start.go (hunk 1 above)
Comment (mattverse) [reply server/start.go:295]: I'm assuming it has something related to the versionings of sdk 0.46, and trying not to break it, but don't have a clear idea in my mind

Comment (p0mvn) [reply server/start.go:295]: Yeah that's correct. It's to avoid having SDK 0.46 app breaking our app API

**Pull request merged**